src/man/authselect-profiles.5.adoc

```diff
@@ -53,7 +53,9 @@ done to the system.
      the modules in the system-auth configuration file._
 
 *nsswitch.conf*::
-    Name Service Switch configuration file.
+    Name Service Switch configuration file. Only maps relevant to the profile
+    must be set. Maps that are not specified by the profile are included from
```

Comment:
Are the relevant maps the same for every profile? If yes, it might be good to list them.

Reply:
No. It depends entirely on the profile.